src/routes/Player/styles.less

```diff
@@ -53,9 +54,39 @@ html:not(.active-slider-within) {
                 color: @color-surface-light5-90;
                 text-align: center;
             }
+
+            .error-details {
+                gap: 0.8rem;
```

Comment:
allways declare a flex property explicitly

Reply:
i meant to allways declare a prop like `flex: XXX` explicitly when parent has `display: flex`

Reply:
there is still usage of `gap` here